androidapp/app/src/main/java/com/example/lanweather/ui/CommonUtils.kt

```diff
@@ -0,0 +1,50 @@
+package com.example.lanweather.ui
+
+import android.content.Context
+import android.content.res.Resources
+import android.widget.ImageView
+import androidx.core.content.res.ResourcesCompat
+import com.example.lanweather.R
+import java.lang.StringBuilder
+
+class CommonUtils {
```

Comment:
https://kotlinlang.org/docs/coding-conventions.html#source-file-organization

naming -- see second paragraph.

Reply:
I'll take a look at that